src/scripts/components/Play.js

```diff
@@ -540,86 +565,93 @@ class Play extends Component<Props, State> {
-              <ShareOverlay show={this.state.showShareModal}>
-                <CloseButton onClick={this.toggleShareModal}>
-                  <SVGButton>
-                    <use xlinkHref="#icon-close" />
-                  </SVGButton>
-                </CloseButton>
-                <ShareTitle small />
-                <AnchorLink
-                  href={
-                    this.getPortalUrl() + '/play/' + ((video && video.id) || '')
-                  }
-                  target="_blank"
-                  anchor
-                  white
-                >
-                  {this.getPortalUrl() + '/play/' + ((video && video.id) || '')}
-                </AnchorLink>
-                <ShareButtons>
-                  <ShareLink
-                    href={this.getTelegramHref()}
-                    target="_blank"
-                    anchor
-                  >
-                    <ShareLinkIcon src="/assets/svg/icons-share-telegram.svg" />
-                  </ShareLink>
-                  <ShareLink
-                    href={this.getTwitterHref()}
-                    target="_blank"
-                    anchor
+          <VideoWrapper isEmbed={isEmbed}>
+            <PlayerWrapper
+              onClick={this.onPlayerClick}
+              onMouseEnter={this.onMouseEnter}
+              innerRef={(ref: HTMLElement) => {
+                this.wrapperRef = ref
+              }}
+            >
+              <Transition in={this.state.shouldShowVideoOverlay} timeout={0}>
+                {(transitionState: ?string) => (
+                  <OverlayWrapper
+                    onMouseLeave={this.onMouseLeave}
+                    onMouseMove={this.onMouseMove}
                   >
-                    <ShareLinkIcon src="/assets/svg/icons-share-twitter.svg" />
-                  </ShareLink>
-                  <ShareLink
-                    href={this.getWhatsAppMobileHref()}
+                    <VideoOverlay
+                      onClick={this.onOverlayClick}
+                      video={video}
+                      isEmbed={isEmbed}
+                      toggleShareModal={this.toggleShareModal}
+                      showShareModal={this.state.showShareModal}
+                      onScrub={this.scrubVideo}
+                      onVolumeChange={this.changeVolume}
+                      onToggleMute={this.toggleMute}
+                      transitionState={transitionState}
+                      togglePlayPause={this.togglePlayPause}
+                      toggleFullscreen={(goToFullscreen: boolean): void => {
+                        if (goToFullscreen && this.wrapperRef) {
+                          requestFullscreen(this.wrapperRef)
+                        } else {
+                          requestCancelFullscreen()
+                        }
+                      }}
+                    />
+                  </OverlayWrapper>
+                )}
+              </Transition>
+              <Player id="player" />
```

Comment:
this `player id="player"` is still there hardcoded? is it correct @bent0b0x?

Reply:
This is needed because of `clappr`. I don't think we have a feasible way around this right now